src/plugins/directory.js

```diff
@@ -0,0 +1,59 @@
+export const classProperties = {
```

Comment:
-legacy will be removed (there's a pr to make it the regular one) https://github.com/babel/babel/issues/5283